src/main/kotlin/creator/LicenseStep.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2021 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.creator
+
+import com.demonwav.mcdev.platform.fabric.creator.FabricTemplate
+import com.demonwav.mcdev.util.License
+import com.intellij.openapi.progress.ProgressIndicator
+import com.intellij.openapi.project.Project
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.StandardOpenOption
+
+class LicenseStep(
+    private val project: Project,
+    private val rootDirectory: Path,
+    private val license: License,
+    private val author: String
+) : CreatorStep {
+
+    override fun runStep(indicator: ProgressIndicator) {
+        val licenseFile = rootDirectory.resolve("LICENSE")
+
+        val fileText = FabricTemplate.applyLicenseTemplate(project, license, author)
```

Comment:
Maybe we should move this from `FabricTemplate` to a `CommonTemplate` object or something like that.